skxray/img_proc/mathops.py

```diff
@@ -0,0 +1,349 @@
+# Module for the BNL image processing project
+# Developed at the NSLS-II, Brookhaven National Laboratory
+# Developed by Gabriel Iltis, Oct. 2013
+"""
+This module is designed to facilitate image arithmetic and logical operations
+on image data sets.
+"""
+
+import numpy as np
+import parser
+
+
+def arithmetic_basic(input_1,

```

Comment:
1. Rename to `arithmetic`
1. Change argument order so that `operation` is first
1. Rename args to `x1`, `x2`
1. Oh, and move this to `vttools/to_wrap/some_module_name.py`